### ISSUE TITLE
Incompatibility with pykeepass

### ISSUE THREAD

**Issue by aivanovski** [keemobile/kotpass#24]
Started as this issue https://github.com/aivanovski/keepassvault/issues/240, there are links to other issues inside, but I'm not sure if they provide some clarity.

It looks like there is some incompatibility with pykeepass library and after any field was inserted into database created by kotpass, kotpass interprets that fields as `EntryValue.Encrypted`. Before modification only password field was `EntryValue.Encrypted`.

Here are steps to reproduce:
1 - Create new database with kotpass (`KeePassDatabase.Ver4x.create` all default parameters), check that target field is `EntryValue.Plain` (use field notes, url, username)
2 - Modify target field with pykeepass
3 - Open database with kotpass, check the type of target field
**Actuals result:** target field is `EntryValue.Encrypted`
**Expected result:** target field is `EntryValue.Plain`

Python code for step 2:
```python
from pykeepass import PyKeePass

kp = PyKeePass('test.kdbx', password='abc123')

entry = kp.root_group.entries[0]
entry.notes = "Notes from pykeepass"

kp.save()
```

And database file, password is `abc123`:
[test.kdbx.zip](https://github.com/keemobile/kotpass/files/15369955/test.kdbx.zip)

**Comment by aivanovski** [comment]
[pykeepass-modified.kdbx.zip](https://github.com/keemobile/kotpass/files/15386665/pykeepass-modified.kdbx.zip)

**Comment by aivanovski** [comment]
This turned out to be a pykeepass problem, this issue was fixed in https://github.com/libkeepass/pykeepass/pull/381
Will be available in pykepass > 4.0.7
